src/main/scala/com/cognite/sdk/scala/v1/resources/assets.scala

```diff
@@ -90,4 +90,12 @@ object Assets {
     deriveEncoder[AssetsQuery]
   implicit val assetsFilterRequestEncoder: Encoder[FilterRequest[AssetsFilter]] =
     deriveEncoder[FilterRequest[AssetsFilter]]
+  implicit val cogniteIdEncoder: Encoder[CogniteId] = new Encoder[CogniteId] {
```

Comment:
Shouldn't have to copy-paste this, add it to common or something and import at necessary? I think we do that for many other encoders.

Reply:
Created a package object for some common Encoders/Decoders, let me know if that's all right.

Reply:
Sounds good to me.